pkg/infinity/csvBackend.go

```diff
@@ -46,7 +47,11 @@ func GetCSVBackendResponse(ctx context.Context, responseString string, query mod
 		frame.Fields = append(frame.Fields, newFrame.Fields...)
 	}
 	if err != nil {
-		err = errorsource.PluginError(err, false)
+		if errors.Is(err, csvframer.ErrEmptyCsv) || errors.Is(err, csvframer.ErrReadingCsvResponse) {
```

Comment:
Is `ErrReadingCsvResponse` always guaranteed to be a downstream error or could it be an error in parsing?

Reply:
Yes, this error is returned for an invalid CSV string that can't be parsed by the CSV standard library, which throws an error for it.